apps/datahub/src/app/record/record-metadata/record-metadata.component.spec.ts

```diff
@@ -176,49 +176,6 @@ describe('RecordMetadataComponent', () => {
         ).toBeFalsy()
       })
     })
-    describe('Image Overlay Preview', () => {
```

Comment:
Did you remove these because we already have e2e tests for it?

Reply:
Image overview is no more in record metadata component, it is now in the header, I have added some minimal UTs, they were missing.